app/Services/Interventions/MaintenanceContractService.php

```
@@ -0,0 +1,186 @@
+        }
+
+        $recipient = $this->resolveRecipient($contract);
+
+        if (blank($recipient)) {
+            return false;
+        }
+
+        Notification::route('mail', $recipient)
+            ->notify(new MaintenanceContractReminderNotification($contract));
+
+        MaintenanceContractReminder::firstOrCreate([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Interventions\MaintenanceContractReminder::firstOrCreate().


(staticMethod.notFound)